library/src_dotc/dotty/DottyPredef.scala

```diff
@@ -0,0 +1,471 @@
+package dotty
```

Comment:
I am not sure about the `src_dotc` scheme or the other aspects of the build. Let's discuss this aspect with the dotty team as a whole.